Lib/fontbakery/profiles/universal.py

```diff
@@ -538,29 +538,31 @@ def com_google_fonts_check_whitespace_ink(ttFont):
 
 @check(
     id='com.google.fonts/check/required_tables',
-    conditions = ['is_ttf'],
+    conditions = ['ttFont'],
     rationale = """
-        Depending on the typeface and coverage of a font, certain tables are
-        recommended for optimum quality.
-        
-        For example:
-        
-        - the performance of a non-linear font is improved if the VDMX, LTSH,
-          and hdmx tables are present.
-          
-        - Non-monospaced Latin fonts should have a kern table.
-        
-        - A gasp table is necessary if a designer wants to influence the sizes
-          at which grayscaling is used under Windows. Etc.
```

Comment:
Please do not remove these, unless you believe there are incorrect statements. Either keep these sentences, or move them somewhere else.

For instance, one possibility would be to have the reasoning for each specific table being displayed as part of the relevant INFO/FAIL messages when such tables are missing.

Reply:
I removed them because they seemed unrelated to the check's goal which is to enforce required tables, and inform about optional tables.
I'm not against putting them back, but I don't think they belong in the check's rationale.
LMK what's the best place for them.

Reply:
Reinstated in 7a650fa7526a1884ff3462836289d94bad063c34

Reply:
The rationale is supposed to explain why something is enforced. These statements fulfill that. If we remove them from rationale, then they should move to log messages.

The `UNWANTED_TABLES` dictionary containing the specific reasoning for each table at **com.google.fonts/check/unwanted_tables** is an example of this alternative approach.

Reply:
Generic comments moved back to rationale in a767d8e192ef277f3778714b75bf3b44bbece62d

Reply:
The only table mentioned by these additional comments that is being enforced by the check is `hdmx`. All the other tables are optional, thus not enforced.

Reply:
Actually, it's `hmtx` that is enforced, not `hdmx`.